samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
All good!

While you're here, erh, would you happen to know if the staff would object to me swapping out a grammar with a more detailed and accurate one?

That was the next PR I was going to submit. The header of [Lightshow](https://github-lightshow.herokuapp.com/) says "If you have an improved grammar we should be using on GitHub, please [file an issue](https://github.com/github/linguist/issues/new)." so I'd vouch the answer is "yes"... but since the grammar in question was written by me, I still feel like a prick. :(